### DISCUSSION TITLE
Upgrading evo 1 controller to current version 1.21

### DISCUSSION THREAD

**Discussion by rhombixt** [jgyates/genmon#1167] [General]
Any issues with genmon and evo 1 controllers being upgraded? I want to upgrade my version to get the remote shutdown switch working, but not sure about what release (if any) generac enabled encryption . Current is 1.21 - just want to confirm it’s ok to do this ?

**Comment by jgyates** [comment]
There are no issues that I am aware of with any firmware version.

**Comment by rhombixt** [comment]
How did you work around this “encryption” generac added or did they even add it to the 1.0 evo controllers ?

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by jgyates** [comment]
This has not been added to evo 1, only evo 2 air cooled units. Genmon handles decryption for evo 2.

**Reply by jgyates** [reply]
Evo 1 does not encrypt the data
